metadata.rb

```diff
@@ -882,14 +882,6 @@
           :description =>  "the version of docker to use on centos installation",
           :type => 'string'
 
-attribute "hops/selinux_version/centos",
-          :description =>  "the version of selinux to use on centos installation",
-          :type => 'string'
-
-attribute "hops/containerd_version/centos",
-          :description =>  "the version of containerd to use on centos installation",
-          :type => 'string'
-
 attribute "hops/containerd_version/ubuntu",
```

Comment:
Can you remove the other variables as well? Like this one, but there are probably more that we are not using anymore.